packages/devtools_app/lib/src/shared/managers/banner_messages.dart

```diff
@@ -23,6 +23,13 @@ const _runInProfileModeDocsUrl = 'https://flutter.dev/to/use-profile-mode';
 const _cpuSamplingRateDocsUrl =
     'https://docs.flutter.dev/tools/devtools/cpu-profiler#cpu-sampling-rate';
 
+/// Screen id to use for banner messages that are intended to be universal for
+/// every DevTools screen.
+///
+/// Messages with this screen id will be added to the list of messages for
+/// every screen from the [BannerMessages] widget.
+const universalBannerMessageId = 'universal';
```

Comment:
Is the banner ID the same as the banner key? (Can there be more than one universal banner? Does dismissing one of them dismiss all of them?)

Reply:
Good question. No this is the ID to use for the screen, not the ID for the banner message itself. There can be more than one banner message with this screen id, much like there can be multiple banner messages for any other screen id. I will rename this to `universalScreenId` to make this less confusing. Banner messages are still identified by their key.